### DISCUSSION TITLE
Cannot connect to backhand

### DISCUSSION THREAD

**Discussion by wassuphelp** [silveredgold/beta-protection#20] [Q&A] [answered]
Hello, I've received a error messege on ghrome saying [2022-12-08T20:32:23.033Z] Error: Failed to start the connection: Error: WebSocket failed to connect. The connection could not be found on the server, either the endpoint may not be a SignalR endpoint, the connection ID is not present on the server, or there is a proxy blocking WebSockets. If you have multiple servers check that sticky sessions are enabled. While in the extension is say's can't connect to backhand even though I entered the backhand adress (http://localhost:2382).
I can use the setting and edit them but pictures will load forever without them ever finishing loading.

**Comment by ghost** [comment]
I'm having the same problem

**Comment by nisaeggu** [comment]
The "backend" must be running in the background in order for the extension to be able to connect

**Comment by silveredgold** [comment, marked as answer]
As mentioned by nisaeggu, you need to make sure you have a compatible censoring backend running, and make sure you set the correct URL for your backend in the settings as per [the usage docs](https://silveredgold.github.io/beta-protection/guide/usage.html#backend-configuration)
